xbmc/powermanagement/PowerManager.cpp

```diff
@@ -256,6 +257,8 @@ void CPowerManager::OnSleep()
 
   CServiceBroker::GetPVRManager().OnSleep();
   g_application.SaveFileState(true);
+  m_lastItemFile.reset(new CFileItem(g_application.CurrentFileItem()));
```

Comment:
I don't know where we did lose the CurrentFile during player termination, I had to create a copy of it to keep it.